src/main/java/com/facebook/openwifirrm/ucentral/UCentralClient.java

```diff
@@ -591,6 +593,71 @@ public SerialNumberList getProvInventoryForRRM() {
 		}
 	}
 
+	/** Retrieve the RRM config and schedule for a specific AP */
```

Comment:
Please use our standard format of java doc, i.e. including the args in the java doc.

Reply:
sounds good - got confused because i've seen a lot that only has the comment but will keep in mind for future